queries/go/highlights.scm

```diff
@@ -88,7 +88,6 @@
   "continue"
   "default"
   "defer"
-  "func"
   "go"
```

Comment:
`"go"` is used to start goroutines. Is it a `@keyword.function`? I don't know.

Reply:
"go" precedes calls, not definitions, so I don't think it is.